examples/all-clusters-app/all-clusters-common/all-clusters-app.zap

```diff
@@ -3365,6 +3365,172 @@
             }
           ]
         },
+        {
+          "name": "Time Sync",
+          "code": 56,
+          "mfgCode": null,
+          "define": "TIME_SYNC_CLUSTER",
+          "side": "client",
+          "enabled": 0,
+          "commands": [
+            {
+              "name": "SetUtcTime",
+              "code": 0,
+              "mfgCode": null,
+              "source": "client",
+              "incoming": 1,
+              "outgoing": 1
+            }
+          ],
+          "attributes": [
+            {
+              "name": "ClusterRevision",
+              "code": 65533,
+              "mfgCode": null,
+              "side": "client",
+              "included": 1,
+              "storageOption": "RAM",
+              "singleton": 1,
+              "bounded": 0,
+              "defaultValue": "2",
+              "reportable": 0,
+              "minInterval": 1,
+              "maxInterval": 65534,
+              "reportableChange": 0
+            }
+          ]
+        },
+        {
+          "name": "Time Sync",
+          "code": 56,
+          "mfgCode": null,
+          "define": "TIME_SYNC_CLUSTER",
+          "side": "server",
+          "enabled": 1,
+          "commands": [],
+          "attributes": [
+            {
+              "name": "UTCTime",
+              "code": 0,
+              "mfgCode": null,
+              "side": "server",
+              "included": 1,
+              "storageOption": "RAM",
+              "singleton": 1,
+              "bounded": 0,
+              "defaultValue": "",
+              "reportable": 0,
+              "minInterval": 1,
+              "maxInterval": 65534,
+              "reportableChange": 0
+            },
+            {
+              "name": "Granularity",
+              "code": 1,
+              "mfgCode": null,
+              "side": "server",
+              "included": 1,
+              "storageOption": "RAM",
+              "singleton": 1,
+              "bounded": 0,
+              "defaultValue": "",
+              "reportable": 0,
+              "minInterval": 1,
+              "maxInterval": 65534,
```

Comment:
If this is max value for the field, it should be either max of enum8 or values of 0-4 (enum values in the spec). Again, sorry, I'm not familiar with the zap conventions here, so I'm not totally clear on what this represents or what the convention for enums is. 

Ditto with the time source below - it's an enum8 with values from 0 to 16.